### DISCUSSION TITLE
Numeric type (custom)

### DISCUSSION THREAD

**Discussion by r-bart** [supabase/supabase#3118] [Questions] [answered]
Hi!

I don't see the option in data types dropdown.

How can I add a custom Numeric type?

I'm specifically aiming to add a uint256 column.

Thank you!

**Comment by soedirgo** [comment, marked as answer]
Hey @raw-brt, I think you'll have to resort to SQL for now:
```sql
ALTER TABLE my_table ADD my_column numeric(78);
```
